Documentation/ApiOverview/UpdateWizards/Concept/Index.rst

```diff
@@ -0,0 +1,33 @@
+.. include:: ../../../Includes.txt
+
+.. _update-wizards-concept:
+
+=============================
+The concept of update wizards
+=============================
+
+Update wizards are single PHP Classes which provide an automated way to update certain
```

Comment:
"[...] single PHP **c**lasses **that** provide [...]"
("classes" lowercase; "that" is better than "which", because something abstract [not a thing, not a person] is referenced)

Reply:
Sorry - as mentioned below, capitalizing "wizards" is nonsense and causes confusion. It would mean that words like "core" should be capitalized, too, and we soon would be in hell's kitchen...
So again: sorry for that! Please revert - thank you!